src/main/java/com/sprint/mission/discodeit/security/CsrfController.java

```diff
@@ -0,0 +1,26 @@
+package com.sprint.mission.discodeit.security;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.web.csrf.CsrfToken;
+import org.springframework.security.web.csrf.CsrfTokenRepository;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+public class CsrfController {
+
+  private final CsrfTokenRepository csrfTokenRepository;
+
+  @GetMapping("/api/auth/csrf-token")
+  public ResponseEntity<CsrfToken> getCsrfToken(HttpServletRequest request,
+      HttpServletResponse response) {
+    request.getSession(true);
+    CsrfToken token = csrfTokenRepository.generateToken(request);
+    csrfTokenRepository.saveToken(token, request, response);
+    return ResponseEntity.ok(token);
+  }
```

Comment:
```suggestion
  @GetMapping("/api/auth/csrf-token")
  public ResponseEntity<CsrfToken> getCsrfToken(CsrfToken token) {
    return ResponseEntity.ok(token);
  }
```
이미 csrf filter 를 거친 상태라 세션에 csrf 가 발급이 되었을거에요!
위처럼 받을 수 있더라구요.

Reply:
오.. 이렇게 하면 스프링이 필터에서 발급받은 CsrfToken 을 자동으로 주입해주는 군요..! 감사합니다!